src/screens/CreatePost/elements/SheetAddTopic.js

```diff
@@ -190,9 +188,7 @@ const SheetAddTopic = ({refTopic, onAdd, topics, onClose, chatTopics, isAnonimit
           <Text style={styles.textDesc}>
             Hit space to finish adding a community. Add up to 5 communities.
           </Text>
-          <Button onPress={() => save()} style={{backgroundColor: dynamicColor.primary}}>
-            Save
-          </Button>
+          <Button onPress={() => save()}>Save</Button>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The button's style has been removed in the new hunk. If the `dynamicColor.primary` was used to adapt the button's color based on the current theme, this change could affect the app's appearance.

```diff
- <Button onPress={() => save()} style={{backgroundColor: dynamicColor.primary}}>
+ <Button onPress={() => save()}>Save</Button>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->